packages/core-scroll/core-scroll.js

```diff
@@ -77,10 +77,22 @@ export default class CoreScroll extends HTMLElement {
     window.addEventListener('load', this) // Update state when we are sure all CSS is loaded
     document.addEventListener('click', this)
     setTimeout(() => this.handleEvent()) // Initialize buttons after children is parsed
+    
+    // Initialize buttons when children change
+    // - jsx relies on onScrollChange triggering to update button states
+    this._childListObserver = new MutationObserver((mutationList) => {
+      for (const mutation of mutationList) {
+        if (mutation.type === 'childList') {
+          this.handleEvent();
+        }
+      }
+    })
+    this._childListObserver.observe(this, { childList: true, subtree: true })
   }
 
   disconnectedCallback () {
     this._throttledEvent = null // Garbage collection
+    this._childListObserver.disconnect();
```

Comment:
Make sure `this._childListObserver` exists before invoking `disconnect()` (example [here](https://github.com/nrkno/core-components/blob/master/packages/core-tabs/core-tabs.js#L23))

Reply:
also, place this above the `// Garbage collection` and set `this._childListObserver` to `null` as well (example [here](https://github.com/nrkno/core-components/blob/master/packages/core-suggest/core-suggest.js#L55))

Reply:
very reasonable 🙌